docs/source/distributed.rst

```diff
@@ -59,15 +59,16 @@ included if you build PyTorch from source. (e.g.building PyTorch on a host that
 installed.)
 
 .. warning ::
```

Comment:
Should we change warning to note here, as we have a backend parity between Windows and Linux?

Reply:
Sure, just resolved.